.github/workflows/sdk-resources-registry-registry-create.yml

```diff
@@ -65,8 +65,10 @@ jobs:
           bash "${{ github.workspace }}/infra/sdk_helpers.sh" generate_workspace_config "../../.azureml/config.json";
           bash "${{ github.workspace }}/infra/sdk_helpers.sh" replace_template_values "registry-create.ipynb";
           [ -f "../../.azureml/config" ] && cat "../../.azureml/config";
-          papermill -k python registry-create.ipynb registry-create.output.ipynb
+          papermill -k python registry-create.ipynb registry-create.output.ipynb;
+          az ml registry delete --name "DemoRegistryPython"-$timestamp --resource-group $RESOURCE_GROUP_NAME --subscription $SUBSCRIPTION_ID
```

Comment:
This change will get overridden on next regeneration of the GitHub workflows.
Please make an appropriate change so that the call to python readme.py won't override this change.

Reply:
Done